### PR TITLE
Application theme integration

### PR DESCRIPTION
**Pull request by mindnervestech**

### CODE REVIEW

File: app/app.controller.js [hunk 1]
```diff
@@ -1,13 +1,63 @@
 "use strict";
 
 angular.module("alu-apply")
-    .controller("ApplicationController", [ "$scope",
-        function( $scope) {
+    // .controller("ApplicationController", [ "$scope",
```
Comment (oolawo09) [inline app/app.controller.js]: Could you remove all commented code?

File: app/app.controller.js [hunk 2]
```diff
@@ -1,13 +1,63 @@
...
+
+ .controller("ApplicationController", ["$scope", "UserApplicationService", "$state", ApplicationController]);
+function ApplicationController($scope, UserApplicationService, $state) {
+	$scope.dasboard = false;
+   	var g = 992;
+   	$scope.toggle = true;
+   	$scope.getWidth = function() {
+     	return window.innerWidth
+   	}, $scope.$watch($scope.getWidth, function(o, n) {
+     // o >= g ? angular.isDefined(e.get("toggle")) ? t.toggle = !!e.get("toggle") : t.toggle = !0 : t.toggle = !1
```
Comment (oolawo09) [inline app/app.controller.js]: remove commented code

File: app/app.controller.js [hunk 3]
```diff
@@ -1,13 +1,63 @@
...
+    //         $scope.setCurrentUser = function(user) {
+    //             $scope.currentUser = user;
+    //         };
+    //     }
+    // ]);
+
+ .controller("ApplicationController", ["$scope", "UserApplicationService", "$state", ApplicationController]);
+function ApplicationController($scope, UserApplicationService, $state) {
+	$scope.dasboard = false;
+   	var g = 992;
```
Comment (oolawo09) [inline app/app.controller.js]: could you give a more descriptive variable name here ?

File: app/app.controller.js [hunk 4]
```diff
@@ -1,13 +1,63 @@
...
-    ]);
+    //         $scope.setCurrentUser = function(user) {
+    //             $scope.currentUser = user;
+    //         };
+    //     }
+    // ]);
+
+ .controller("ApplicationController", ["$scope", "UserApplicationService", "$state", ApplicationController]);
+function ApplicationController($scope, UserApplicationService, $state) {
+	$scope.dasboard = false;
```
Comment (oolawo09) [inline app/app.controller.js]: Could you take controller variables that aren't accessed outside the controller off of the $scope variable and attach them to the controller directly? 

For example $scope.dashboard would become 
 
var vm = this;
vm.dashboard;

File: app/app.controller.js [hunk 5]
```diff
@@ -1,13 +1,63 @@
...
+   	var g = 992;
+   	$scope.toggle = true;
+   	$scope.getWidth = function() {
```
Comment (oolawo09) [inline app/app.controller.js]: Could you take controller functions that aren't accessed outside the controller off of the $scope variable and attach them to the controller directly? 

For example $scope.getWidth would become 
 
var vm = this;
vm.getWidth;

File: app/app.controller.js [hunk 6]
```diff
@@ -1,13 +1,63 @@
...
+     	return window.innerWidth
+   	}, $scope.$watch($scope.getWidth, function(o, n) {
+     // o >= g ? angular.isDefined(e.get("toggle")) ? t.toggle = !!e.get("toggle") : t.toggle = !0 : t.toggle = !1
+   	}), $scope.toggleSidebar = function() {
+      $scope.toggle = !$scope.toggle
```
Comment (oolawo09) [inline app/app.controller.js]: Could you take controller functions that aren't accessed outside the controller off of the $scope variable and attach them to the controller directly? 

For example $scope.getWidth would become 
 
var vm = this;
vm.getWidth;

File: app/userApplication/userApplication.template.html [hunk 7]
```diff
@@ -0,0 +1,12 @@
+<link rel="stylesheet" href="bower_components/angular/angular-csp.css">
+<link rel="stylesheet" href="bower_components/font-awesome/css/font-awesome.min.css">
+<link rel="stylesheet" href="bower_components/angular-material/angular-material.min.css">
+<link rel="stylesheet" href="bower_components/angular-surveys/dist/form-builder-material.min.css">
```
Comment (oolawo09) [inline app/userApplication/userApplication.template.html]: I don't think you need this if you're using the formViewer

File: app/taskGeneration/taskGeneration.template.html [hunk 8]
```diff
@@ -0,0 +1,28 @@
+<link rel="stylesheet" href="taskGeneration/taskGeneration.css">
+<toast></toast>
```
Comment (oolawo09) [inline app/taskGeneration/taskGeneration.template.html]: What does <toast> do ?

File: app/taskGeneration/taskGeneration.controller.js [hunk 9]
```diff
@@ -0,0 +1,103 @@
+"use strict";
+
+agGrid.initialiseAgGridWithAngular1(angular);
+angular.module("taskGeneration")
+    .controller("TaskGenerationController", ['$http', '$scope', '$state', '$stateParams', '$localStorage', 'ngToast', 'TaskGenerationService', function($http, $scope, $state, $stateParams, $localStorage, ngToast, TaskGenerationService) {
+          var vm = this;
+
+      		//list all tasks available
```
Comment (oolawo09) [inline app/taskGeneration/taskGeneration.controller.js]: You can get rid of this if you're not using it

File: app/stageGeneration/stageGeneration.template.html [hunk 10]
```diff
@@ -1,37 +1,72 @@
 <link rel="stylesheet" href="stageGeneration/stageGeneration.css">
-<div class="container">
+<toast></toast>
```
Comment (oolawo09) [inline app/stageGeneration/stageGeneration.template.html]: What does this do ?

File: app/app.controller.js [hunk 11]
```diff
@@ -1,13 +1,63 @@
...
+    // ]);
+
+ .controller("ApplicationController", ["$scope", "UserApplicationService", "$state", ApplicationController]);
+function ApplicationController($scope, UserApplicationService, $state) {
+	$scope.dasboard = false;
+   	var g = 992;
+   	$scope.toggle = true;
+   	$scope.getWidth = function() {
+     	return window.innerWidth
+   	}, $scope.$watch($scope.getWidth, function(o, n) {
```
Comment (oolawo09) [inline app/app.controller.js]: this looks like a syntax error.

File: app/app.controller.js (hunk 4 above)
Comment (oolawo09) [inline app/app.controller.js]: Could you attach this to the vm variable using the controllerAs syntax ?

File: app/stageGeneration/stageGeneration.template.html (hunk 10 above)
Comment (mindnervestech) [inline app/stageGeneration/stageGeneration.template.html]: This is showing the toast messages - 
 - Success messages (Task has been created successfully),
 - Warning messages (Task name should be unique) ,
 -  Info messages  (Task has been deleted)
we are using ng-toast in stage generation , application generation , task generation , form generation and user application

File: app/userApplication/userApplication.template.html (hunk 7 above)
Comment (mindnervestech) [inline app/userApplication/userApplication.template.html]: some of file i need to use for  formViewer because some css styles not applying to form

**Comment by oolawo09** [comment]
Could you also align/indent all the code ?

**Pull request closed without merging**